.github/workflows/setup-and-test.yml

```diff
@@ -28,6 +31,38 @@ jobs:
       - name: Run tests
         run: CI=true yarn test
 
+      - name: Get the git SHA and use it as the repo-branch
+        id: git-sha
+        run: |
+          CONSUMER_VERSION=$(git rev-parse --abbrev-ref HEAD)
+          echo "repo-branch=${CONSUMER_VERSION}" >> $GITHUB_OUTPUT
+          echo $CONSUMER_VERSION
+
+      - name: Encode the pact as non-breaking base 64 string
+        id: encode
+        env:
+          PACT_FULL_PATH: 'pacts/cbas-ui-cbas.json'  # this is a temporary hack until WM-1858 is done.
```

Comment:
It might be good idea to create a ticket to update this when WM-1858 is done (unless it already exists then please ignore my comment)